revscoring/datasources/tests/test_revision.py

```diff
@@ -21,7 +22,6 @@ def test_content():
     eq_(solve(revision.content_tokens, cache=cache),
         ["This", " ", "is", " ", "a", " ", "foobar", "  ", "<td>"])
 
-
```

Comment:
This part is not PEP8 compliment, before any first-order function definition should be two empty lines